diagram.md

```diff
@@ -252,7 +252,7 @@ The `Attachments Element` is for attaching files to a Matroska file such as pict
 ```
 Figure: Representation of a `Attachments Element`.
 
-The `Tags Element` contains metadata that describes the `Segment` and potentially its `Tracks`, `Chapters`, and `Attachments`. Each Track or Chapter that those tags applies to has its UID listed in the tags. The Tags contain all extra information about the file, script writer, singer, actors, directors, titles, edition, price, dates, genre, comments, etc. And it allows you to enter many of these (title, edition, comments, etc.) in different languages.
+The `Tags Element` contains metadata that describes the `Segment` and potentially its `Tracks`, `Chapters`, and `Attachments`. Each `Track` or `Chapter` that those tags applies to has its UID listed in the `Tags`. The `Tags` contain all extra information about the file: scriptwriter, singer, actors, directors, titles, edition, price, dates, genre, comments, etc. It allows you to enter many of these (title, edition, comments, etc.) in different languages.
```

Comment:
> It allows you to…

Maybe change the personal form of address to something impersonal such as "the author". Sounds more RFCy.